src/main/java/com/ldtteam/structurize/storage/rendering/RenderingCache.java

```diff
@@ -122,15 +125,15 @@ public static BlueprintPreviewData getOrCreateBlueprintPreviewData(final String
      */
     public static boolean forceLightLevel()
     {
-        return true;
+        return Structurize.getConfig().getClient().rendererLightLevel.get() >= 0;
     }
 
     /**
      * @return static light level
      */
     public static int getOurLightLevel()
     {
-        return 15;
+        return Structurize.getConfig().getClient().rendererLightLevel.get();
```

Comment:
I think having this in the client config is fine, Im not sure I would have it in the build tool config.

Reply:
it's heavily related tho

Reply:
Yeah, the thing is that I would call this more of a "special" setting, rather than sth that the avg user would touch

Reply:
can add nested settings and move also worldgen/fluid placeholder there, tho i would leave air placeholders in the main one cuz it has more impact

Reply:
So, I think the worldgen/fluid ones is nice to have, but maybe we could have a common setting for that?

Reply:
u mean like merge air/solid/fluid placeholders into "render placeholders nice" which would hide air/render fluid/render worldgen?

Reply:
Ye

Reply:
Merged into one, there are only four now, which looks fine from ui perspective (better than six :D )